coordinator/coordinator.go

```diff
@@ -182,7 +182,9 @@ func GetLockKey(jobKey string) string {
 }
 
 func (k *KubernetesJobSpawner) GetJobRunner(jobName string, config runner.Config, etcdEndpoints []string, id metadata.ResourceID) (types.Runner, error) {
-	etcdConfig := &ETCDConfig{Endpoints: etcdEndpoints, Username: os.Getenv("ETCD_USERNAME"), Password: os.Getenv("ETCD_PASSWORD")}
+	etcd_username := help.GetEnv("ETCD_USERNAME", "root")
```

Comment:
Not sure the best place to add these:
Other options are:

- in values.yaml for the coordinator?
- in the values.yaml in global?
(which means in the deployment.yaml)